### ISSUE TITLE
Standing Laser field

### ISSUE THREAD

**Issue by Danimob** [BEEmod/BEE2.4#1108]

**Comment by Luke18033** [comment]
See [this comment](https://github.com/BEEmod/BEE2-items/issues/257#issuecomment-281786349) and the upcoming features list. You're supposed to be able to connect other fizzler items to it to change the type, but that functionality hasn't been implemented yet.

**Comment by Luke18033** [comment]
@TeamSpen210 Are you working on this?

**Comment by TeamSpen210** [comment]
Sorta, it's on hold till I fix something else. Though I should probably just do it anyway.

**Comment by Luke18033** [comment]
What's the other thing you need to fix?

**Comment by TeamSpen210** [comment]
It's the geometry system, which allows seeing the interior of the fizzler inside the wall. It's complicated to figure out an elegant way to deal with the fact that only the original fizzler/laserfield need a special model for the standing variant.

**Comment by LautaroL20** [comment]
The big problem for me about the feature of changing the type of field of my custom fizzlers is, if this function ever gets made, first of all it must keep using instances as it is now, but also for each type of field so that way it can keep using all the anti-exploit systems ive put with hammer, as well as hundreds of very specific syntax conditions on their `vbsp_config` files , which is something that has cost me so much time (months) of development (understanding the conditions and lot of playtesting) effort that i do not want to throw in the garbage. I do not want this to get break, because TeamSpen210 mentioned the past year on this subject that he would do it another way i was not agreed, because it leaves the instances of my item useless and i don´t want this to happen at all.....

So, for this basically it requires that @TeamSpen210 would guarantee my requirement that I mention above (keep the instances working and the aditional ones for each type of field) which for me it is extremely important, because ive spent a lot of effort doing the item and therefore i know what stuff must never get touched to keep all the features that my item currently have, and keep with their quality that it was conceived for.

**Comment by TeamSpen210** [comment]
The trouble is, the anti-exploit systems I don't think should actually be there. You should be able to fire a portal between the fizzler and adjacent surfaces if that side is left open. There's no way for us to know exactly what the author wanted. Ideally for items I'm wanting to remove specific instances because it means that the item works in every style with every fizzler, instead of having to special-case each of those combinations.

For the conditions, which are those? The portalability selection, choosing whether to create the triangles at the base, and gridded wall generation? For the latter, I've replaced that system with a better one in `geometry` where each ¼ tile section is set independently, and you can do it by just placing a template-entity at every point you want to convert. The former two I think I  have a different way of doing them, which simplifies the logic.

**Comment by LautaroL20** [comment]
> TeamSpen210 Said: "For the conditions, which are those? The portalability selection, choosing whether to create the triangles at the base, and gridded wall generation? For the latter, I've replaced that system with a better one in `geometry` where each ¼ tile section is set independently, and you can do it by just placing a template-entity at every point you want to convert. The former two I think I  have a different way of doing them, which simplifies the logic."

Yes, the current conditions that my item uses are indeed: the pillar´s portalability that are being excecuted thru some conditions plus some "Colorpikers" template entity and choosing when to generate the pillars for the many different situations, depending if the boxel is hollowed or if it gets resized to X2 or X3 thru the buttontype property and also the gridded wall offsets.

I am glad you´re making a new template entity that will take care of making the appropiate gridded type on the wall, but still we need together to supervise how we will simplify the configuration files without degrading how the pillars currently gets generated and their portalability.
